fighthealthinsurance/ml/ml_models.py

```diff
@@ -277,26 +277,28 @@ def __init__(
         self._expensive = expensive
 
     def get_system_prompts(
-        self, prompt_type: str, for_patient: bool = True
+        self, prompt_type: str, prof_pov=False
     ) -> list[str]:
         """
         Get the appropriate system prompt based on type and audience.
 
         Args:
             prompt_type: The type of prompt to get (e.g., 'full', 'questions', etc.)
-            for_patient: Whether this is for patient (True) or professional (False)
+            prof_pov: Generating the appeal from the patient (False) or professional (True) point of view 
 
         Returns:
             The system prompt as a string, or the first prompt if multiple are available
         """
         key = prompt_type
-        if not for_patient and f"{prompt_type}_not_patient" in self.system_prompts_map:
+        prompt = "Your are a helpful assistant with extensive medical knowledge who loves helping patients." 
+        if prof_pov or (prof_pov and f"{prompt_type}_not_patient" in self.system_prompts_map):
             key = f"{prompt_type}_not_patient"
-
+            prompt = "You are a medical expert writing on behalf of a patient. Write as the healthcare professional in first person (“I”)—advocating clearly, clinically, and persuasively for the patient’s medical needs. Emphasize medical necessity, clinical evidence, and patient benefit. Maintain a professional, authoritative tone and keep the focus on the appeal." 
```

Comment:
Gotcha, so maybe add a comment about about switching the default prompt when in patient POV